pkg/types/validation/installconfig.go

```diff
@@ -103,21 +108,28 @@ func validateClusterNetwork(cn *netopv1.ClusterNetwork, fldPath *field.Path, ser
 	return allErrs
 }
 
-func validateMachinePools(pools []types.MachinePool, fldPath *field.Path, platform string) field.ErrorList {
+func validateControlPlane(pool *types.MachinePool, fldPath *field.Path, platform string) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if pool.Name != "master" {
+		allErrs = append(allErrs, field.NotSupported(fldPath.Child("name"), pool.Name, []string{"master"}))
+	}
+	allErrs = append(allErrs, ValidateMachinePool(pool, fldPath, platform)...)
```

Comment:
We can extract the current "number of replicas must be positive" check from `ValidateMachinePool` and put it in this control-plane-specific validator instead.  And for workers, we have a check in `validateCompute` asserting that at least one `replicas` is strictly positive (the rest can be zero without causing issues).

Reply:
Sure. We can do that once we support more than a single machine pool named "worker".

Reply:
Done.